java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1010,6 +1011,82 @@ public void lrem_existing_non_existing_key_and_type_error(BaseClient client) {
         assertEquals(0, client.lrem("non_existing_key", 0, "value").get());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void lpos_with_options(BaseClient client) {
+        String key = "{ListKey}-1-" + UUID.randomUUID();
+        String[] valueArray = new String[] {"a", "a", "b", "c", "a", "b"};
+        assertEquals(6L, client.rpush(key, valueArray).get());
+
+        // simplest case
+        assertEquals(0L, client.lpos(key, "a").get());
+        assertEquals(5L, client.lpos(key, "b", LPosOptions.builder().rank(2L).build()).get());
+
+        // element not in the list
+        assertArrayEquals(new Long[] {}, client.lposCount(key, "doesn't_exist", 0L).get());
```

Comment:
Looks like lposCount is tested in the other test, and we're already passing lpos a non-existing element below, so we can probably remove this
```suggestion
```